sourcery_analytics/metrics/utils.py

```diff
@@ -38,12 +38,30 @@ def method_name(method: astroid.nodes.FunctionDef) -> str:
 @nodedispatch
 @validate_node_type(astroid.nodes.FunctionDef)
 def method_lineno(method: astroid.nodes.FunctionDef) -> int:
+    """Returns the line number of the method.
```

Comment:
**suggestion (docstrings):** Please update the docstring for function: `method_lineno`

Reason for update: Addition of examples and a more detailed description.

Suggested new docstring:
```
"""Returns the line number of the method.

Not very useful by itself, but can be combined with other metrics for convenience.

Examples:
    >>> method_lineno("def foo(): pass")
    1
"""
```